canonical-websites/www.ubuntu.com
# Update vanilla v1.6.0

## Done
Updated to Vanilla-framework@1.6.0. Checked through the site and removed redundant code. Also added a few hotfixes for Vanilla which should all have issues as comments.

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Look through the site and make sure it looks ok
